src/main/java/com/github/splendor_mobile_game/websocket/handlers/reactions/StartGame.java

```diff
@@ -16,65 +41,209 @@
  *      "messageContextId": "02442d1b-2095-4aaa-9db1-0dae99d88e03",
  *      "type": "START_GAME",
  *      "data": {
- *          "userUuid": "6850e6c1-6f1d-48c6-a412-52b39225ded7"
+ *          "userDTO":{
+ *              "uuid": "6850e6c1-6f1d-48c6-a412-52b39225ded7",
+ *              "name":"James"
+ *          },
+ *          "roomDTO":{
+ *              "uuid": "6850e6c1-6f1d-48c6-a412-52b39225ded7"
```

Comment:
I think host should not be obligated to send their name. It's not used actually.